examples/with-flow/types/next.js.flow

```diff
@@ -0,0 +1,41 @@
+declare module "next" {
+  declare type NextApp = {
+    prepare(): Promise<void>;
```

Comment:
@nkzawa 
I added them and `getRequestHandler` for custom-handler. For https://github.com/zeit/next.js/blob/master/examples/custom-server/server.js#L7